apps/aeoracle/src/aeo_query_tx.erl

```diff
@@ -244,9 +242,14 @@ serialization_template(?ORACLE_QUERY_TX_VSN) ->
 version(_) ->
     ?ORACLE_QUERY_TX_VSN.
 
+
 -spec valid_at_protocol(aec_hard_forks:protocol_vsn(), tx()) -> boolean().
-valid_at_protocol(_, _) ->
-    true.
+valid_at_protocol(Proto, Tx) ->
+    case {Proto >= ?LIMA_PROTOCOL_VSN, aeser_id:specialize_type(oracle_id(Tx))} of
```

Comment:
😱 This should be valid only Post Lima!

Reply:
This will have to be changed in tests too... It's there for development at the moment.